test/integration/submission/submission.spec.ts

```diff
@@ -755,6 +755,36 @@ describe('Submission Api', () => {
     });
   });
 
+  describe('icgc-import', function() {
+    this.beforeEach(async () => {
+      await clearCollections(dburl, ['donors', 'counters']);
+    });
+
+    it.only('should import legacy samples file', async () => {
+      let file: Buffer;
+      let rows: any[];
+      try {
+        file = fs.readFileSync(stubFilesDir + `/paca.icgc.samples.tsv`);
+        rows = (await TsvUtils.tsvToJson(stubFilesDir + `/paca.icgc.samples.tsv`)) as any;
```

Comment:
this test seems incomplete :stuck_out_tongue:. Shouldn't it also be calling the `/submission/icgc-import/` endpoint with the returned rows from the preprocess endpoint. Or was it just testing the precrocess returns 200...

Reply:
yeah this test just calls preprocess